src/main/resources/messages/messages_ru.yml

```diff
@@ -1,106 +1,115 @@
+usage_email_add: '&cИспользование: /email add <эл. почта> <повтор эл. почты>'
+usage_email_change: '&cИспользование: /email change <эл. почта> <новая эл. почта>'
+usage_email_recovery: '&cИспользование: /email recovery <эл. почта>'
+new_email_invalid: '&cНедействительная новая электронная почта!'
+old_email_invalid: '&cНедействительная старая электронная почта!'
+email_invalid: '&cНедействительный адрес электронной почты!'
+email_added: '&2Электронная почта успешно добавлена!'
+email_confirm: '&cПодтвердите свою электронную почту!'
+email_changed: '&2Адрес электронной почты изменён!'
+email_send: '&2Письмо с инструкциями для восстановления было отправлено на вашу электронную почту!'
+email_show: '&2Текущий адрес электронной почты — &f%email'
+incomplete_email_settings: 'Ошибка: не все необходимые параметры установлены для отправки электронной почты. Свяжитесь с администратором.'
+email_already_used: '&4Эта электронная почта уже используется.'
+email_send_failure: 'Письмо не может быть отправлено. Свяжитесь в администратором.'
+show_no_email: '&2К вашей уч. записи не привязана электронная почта.'
+add_email: '&3Добавьте электронную почту: /email add <эл. почта> <повтор эл. почты>'
+recovery_email: '&Забыли пароль? Используйте «/email recovery <эл. почта>».'
+change_password_expired: 'Больше нельзя сменить свой пароль, используя эту команду.'
+email_cooldown_error: '&cПисьмо было отправлено недавно. Подождите %time, прежде чем отправить новое.'
 
 # Каптча
-usage_captcha: '&cВы должны ввести код. Используйте следующую команду: &e/captcha <theCaptcha>'
-wrong_captcha: '&cНеверный код! Используйте следующую команду: &e/captcha THE_CAPTCHA'
-valid_captcha: '&2Вы успешно ввели код!'
+usage_captcha: '&3Необходимо ввести текст с каптчи. Используйте «/captcha <theCaptcha>»'
+wrong_captcha: '&cНеверно! Используйте «/captcha THE_CAPTCHA».'
+valid_captcha: '&2Вы успешно решили каптчу!'
+
+# Код подтверждения
+verification_code_required: '&3Эта команда чувствительна и требует подтверждения электронной почты! Проверьте свою почту и следуйте инструкциям в письме.'
+usage_verification_code: '&cИспользование: /verification <код>'
+incorrect_verification_code: '&cНеверный код, используйте «/verification <код>», подставив код из полученного письма.'
+verification_code_verified: '&2Ваша личность подтверждена! Теперь можно выполнять все чувствительные команды в текущем сеансе!'
+verification_code_already_verified: '&2Вы уже можете выполнять все чувствительные команды в текущем сеансе!'
+verification_code_expired: '&3Срок действия кода истёк! Выполните чувствительную команду, чтобы получить новый код!'
+verification_code_email_needed: '&3Чтобы подтвердить вашу личность, необходимо привязать электронную почту к учётной записи!!'
 
 # Единицы времени
-second: 'сек.'
-seconds: 'сек.'
-minute: 'мин.'
-minutes: 'мин.'
-hour: 'ч.'
-hours: 'ч.'
-day: 'дн.'
-days: 'дн.'
+second: 'с'
+seconds: 'с'
```

Comment:
Is it really smart to change from "sek" to "s" seeing as it may be confused with the word (_with_)? Not a Russian speaker so cannot judge. 
@sgdc3 could you please tag another Russian translator? I'm on mobile for the next day; sorry for the trouble xP

Reply:
Hi @DardBrinza @Bodyash

Reply:
https://ru.wikipedia.org/wiki/%D0%A1%D0%B5%D0%BA%D1%83%D0%BD%D0%B4%D0%B0
«Секунда (русское обозначение: с; международное: s)»

Reply:
Ok, thanks for the reply :)